client/restclient/restclient.go

```diff
@@ -235,8 +244,8 @@ func HandleResponse(response *http.Response, requestStr string) (responseData []
 		if err != nil {
```

Comment:
This condition is never true

Reply:
yes so removed

Reply:
https://github.com/banyansecurity/terraform-provider-banyan/pull/155/commits/afa521abd870a6516d9b06b4d137b7bd9a8d066f#diff-5ec9649c5426a6d1aac77b9e75d2068d03eadd6e98abaa3a16100dfb9784d3fc